examples/cc2538-common/Makefile

```diff
@@ -0,0 +1,11 @@
+CONTIKI_PROJECT = test-pwm timer-test
```

Comment:
`timer-test` uses rtimers, so it needs the original `project-conf.h` to avoid using the default conflicting `NETSTACK_CONF_RDC`.